dts/common/yaml/i2c.yaml

```diff
@@ -0,0 +1,25 @@
+---
+title: I2C Base Structure
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives the base structures for all I2C devices
+
+properties:
+  - clock-frequency :
```

Comment:
I used the clock-frequency because I followed the linux kernel DT bindings for I2C. Moreover, I have read in a previous comment (on github or mailing list, I cannot remember) that ST plans to use the same dts files for linux and zephyr.